packages/react-reconciler/src/ReactChildFiber.js

```diff
@@ -364,6 +404,15 @@ function warnOnSymbolType(returnFiber: Fiber, invalidChild: symbol) {
   }
 }
 
+function warnOnSymbolType(returnFiber: Fiber, invalidChild: symbol) {
+  const debugTask = getCurrentDebugTask();
+  if (__DEV__ && debugTask !== null) {
+    debugTask.run(warnOnSymbolTypeImpl.bind(null, returnFiber, invalidChild));
```

Comment:
Technically these should probably use this technique instead of creating a fake Fiber that gets logged:

https://github.com/facebook/react/blob/main/packages/react-reconciler/src/ReactChildFiber.js#L224-L234

So that `captureOwnerStack` inside `console.error` can also observe it.

It doesn't matter for the throws since Error constructor doesn't observe it anyway.